src/System.Console/tests/Performance/Perf.Console.cs

```diff
@@ -85,5 +84,77 @@ public void OpenStandardError()
                     s.Dispose();
             }
         }
+
+        [Benchmark]
+        [PlatformSpecific(PlatformID.Windows)]
```

Comment:
I was concerned that doing ResetColor may overwrite the user-set console color with the default console color. For example if the user console was set to red but the program default is black, running the tests would change the console to black if we used ResetColor.

After looking at the code again it seems like the colorinfo used in ResetColor is actually set whenever we call the setters of Foreground/Background, so my concerns weren't valid. I'll modify these tests to use ResetColor and be xplat capable, thanks.